directives/submit.py

```diff
@@ -81,6 +83,8 @@ def run(self):
             if 'url' in self.options:
                 data['url'] = self.options['url']
             if 'lti' in self.options:
+                if 'lti1p3' in self.options:
```

Comment:
Should we also check that `lti_custom` is not used with `lti` and `lti_aplus_get_and_post` is not used with `lti1p3`? What about `lti_context_id` and `lti_resource_link_id`?